src/rgw/rgw_rest_log.cc

```diff
@@ -917,7 +921,8 @@ void RGWOp_MDLog_Status::send_response()
 
 // not in header to avoid pulling in rgw_data_sync.h
 class RGWOp_BILog_Status : public RGWRESTOp {
```

Comment:
@adamemerson could you please split the changes in: `RGWOp_BILog_Status` to a separate PR?
I need similar changes in: https://github.com/ceph/ceph/pull/41538/

(see @cbodley comment: https://github.com/ceph/ceph/pull/41538/files/f218891701a933285270e32c117c999e31712dc4#r639101650)

Reply:
Done!